bilby/gw/result.py

```diff
@@ -476,8 +476,8 @@ def plot_interferometer_waveform_posterior(
 
         fd_waveforms = list()
         td_waveforms = list()
-        for _, params in samples.iterrows():
-            params = dict(params)
+        for params in samples.itertuples(index=False):
+            params = dict(params._asdict())
```

Comment:
I think `dict` is redundant as of version Python 3.8 here, see [the docs here](https://docs.python.org/3/library/collections.html#collections.somenamedtuple._asdict).

Reply:
Ah, that's much neater.

Reply:
@foocheng could you update the PR to use Colm's suggestion?

Reply:
Great, thanks!